backend/src/main/java/org/eclipse/tractusx/puris/backend/DataInjectionCommandLineRunner.java

```diff
@@ -166,19 +194,22 @@ private void setupCustomerRole() throws JsonProcessingException {
         // Create PartnerProductStock
         semiconductorMaterial = materialService.findByOwnMaterialNumber(semiconductorMaterial.getOwnMaterialNumber());
         PartnerProductStock partnerProductStockEntity = new PartnerProductStock(
-                semiconductorMaterial,
-                10,
-                supplierPartner.getSiteBpns(),
-                new Date(),
-                supplierPartner
+            semiconductorMaterial,
+            10,
+            supplierPartner.getSites().stream().findAny().get().getBpns(),
```

Comment:
Currently we only support one site, don't we? How about findFirst instead of findAny? Same would apply for all usages.

Reply:
This is really just a shortcut to somehow make this sample in the DataInjectionCommandLineRunner work. I don´t think it´s sensible to assume anything in general here.

Reply:
same as above